drops/azure_edge_extensions_rag_on_edge.json

```diff
@@ -0,0 +1,27 @@
+{
+  "Title": "Retrieval-Augmented Generation on Edge with Azure Arc & Azure IoT Operations",
+  "Summary": "This Drop provides a Retrieval-Augmented Generation(RAG) on edge pattern that facilitates text searches within files utilizing a local vector-based approach Chroma DB and an edge Small/Large Language Model.",

```

Comment:
What about Azure IoT Operations? ARc? Please include this as part of the summary

Reply:
@fcabrera23 This pattern is based on Azure IoT Operations MQ broker for the RAG module messaging. and running on arc-enabled k8s edge env. Do you mean I will need to mention that this pattern is using AIO, ARC in the "summary" section?

Reply:
@fcabrera23 I've added how this pattern makes use of Arc and AIO in the summary